Problem: cannot build non-64 bit binary

```
   • building binaries
      • building                  binary=/home/runner/work/chain-main/chain-main/dist/chain-maincli_windows_386/chain-maincli.exe
      • building                  binary=/home/runner/work/chain-main/chain-main/dist/chain-maincli_darwin_amd64/chain-maincli
      • building                  binary=/home/runner/work/chain-main/chain-main/dist/chain-maincli_linux_amd64/chain-maincli
      • building                  binary=/home/runner/work/chain-main/chain-main/dist/chain-maincli_linux_386/chain-maincli
      • building                  binary=/home/runner/work/chain-main/chain-main/dist/chain-maincli_windows_amd64/chain-maincli.exe
   ⨯ release failed after 201.92s error=failed to build for windows_386: # github.com/tendermint/iavl
##[error]../../../go/pkg/mod/github.com/tendermint/iavl@v0.14.0/nodedb.go:232:65: constant 9223372036854775807 overflows int
##[error]../../../go/pkg/mod/github.com/tendermint/iavl@v0.14.0/repair.go:44:69: constant 9223372036854775807 overflows int
```

Need to remove `386` from .goreleaser.yml

--- a/.github/workflows/goreleaser.yml
+++ b/.github/workflows/goreleaser.yml
@@ -14,12 +14,7 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.14.8
-
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
+      - uses: actions/checkout@v2
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -4,7 +4,7 @@ before:
 
 builds:
   - main: ./cmd/chain-maincli
-    id: "chain-maincli"
+    id: chain-maincli
     binary: chain-maincli
     env:
       - CGO_ENABLED=0
@@ -12,9 +12,8 @@ builds:
       - linux
       - darwin
       - windows
-    ignore:
-      - goos: darwin
-        goarch: 386
+    goarch:
+      - amd64
     ldflags:
       - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=crypto-com-chain -X github.com/cosmos/cosmos-sdk/version.ServerName=chain-maind -X github.com/cosmos/cosmos-sdk/version.ClientName=chain-maincli -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
   - main: ./cmd/chain-maind
@@ -26,23 +25,24 @@ builds:
       - linux
       - darwin
       - windows
-    ignore:
-      - goos: darwin
-        goarch: 386
+    goarch:
+      - amd64
     ldflags:
       - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=crypto-com-chain -X github.com/cosmos/cosmos-sdk/version.ServerName=chain-maind -X github.com/cosmos/cosmos-sdk/version.ClientName=chain-maincli -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
 
 archives:
-- name_template: '{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+- name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
   replacements:
     darwin: Darwin
     linux: Linux
     windows: Windows
-    386: i386
     amd64: x86_64
   format_overrides:
     - goos: windows
       format: zip
+  builds:
+  - chain-maincli
+  - chain-maind
 checksum:
   name_template: 'checksums.txt'
 changelog:
@@ -51,8 +51,8 @@ changelog:
     exclude:
     - '^docs:'
     - '^test:'
-# snapshot:
-#   name_template: "{{ .Tag }}-next"
+snapshot:
+  name_template: "{{ .Tag }}-next"
 
 # brews:
 #   - name: "chain-maind"

